IndoorMapsBackend/src/resolvers/AreaResolver.ts

```diff
@@ -0,0 +1,140 @@
+import { InputType, Field, Int, Resolver, Mutation, Arg, Ctx, Query } from "type-graphql"
+import { getUserOrThrowError } from "../auth/validateUser.js"
+import { Context } from "../utils/context.js"
+import { Area, NewAreaResult } from "../graphqlSchemaTypes/Area.js"
+import { convertToGraphQlArea } from "../utils/typeConversions.js"
+import { GraphQLError } from "graphql"
+
+@InputType()
+class AreaCreateInput {
+    @Field(type => Int)
+    floorDatabseId: number
```

Comment:
Does the previous issue of having unique key across schema and using string keys was resolved?